Framework/DataHandling/inc/MantidDataHandling/LoadSampleShape.h

```diff
@@ -0,0 +1,72 @@
+#ifndef DATAHANDLING_LOAD_SHAPE_H_
+#define DATAHANDLING_LOAD_SHAPE_H_
+
+#include "MantidAPI/IFileLoader.h"
```

Comment:
Why are we still including IFileLoader?

Reply:
I find I need it for `getline` to make sense. There may be a better way of doing this.